src/xrpld/app/consensus/RCLConsensus.cpp

```diff
@@ -92,7 +92,9 @@ RCLConsensus::Adaptor::Adaptor(
               std::numeric_limits<std::uint64_t>::max() - 1))
     , nUnlVote_(validatorKeys_.nodeID, j_)
 {
-    assert(valCookie_ != 0);
+    ASSERT(
```

Comment:
Shouldn't this file have a `#include <xrpl/beast/utility/instrumentation.h>` statement? I presume it compiles due to transitive includes, but it would be better for the file to fully mention all dependencies as imports.

I see this across many other files too, so please double-check all.

Reply:
great point.